homeassistant/components/google_cloud/tts.py

```diff
@@ -281,7 +281,7 @@ async def async_get_tts_audio(self, message, language, options=None):
             )
             # pylint: enable=no-member
 
-            with async_timeout.timeout(10, loop=self.hass.loop):
+            with async_timeout.timeout(10):
```

Comment:
Note that _any_ external library that we use that also have `loop` defined now will fail

Reply:
We will probably need to monkey patch it and warn

Reply:
Sounds good, but I think `2022.2` might be too short, I fear that this is quite common.

Reply:
We can always extend the deadline later.

```
2021-11-03 09:22:08 WARNING (MainThread) [homeassistant.helpers.frame] Detected integration that called async_timeout.timeout with loop keyword argument. The loop keyword argument is deprecated and calls will fail after Home Assistant 2022.2. Please report issue for hunterdouglas_powerview using this method at homeassistant/components/hunterdouglas_powerview/__init__.py, line 102: shade_entries = await shades.get_resources()
```